data/transactions/exec.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2019 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package transactions
+
+import (
+	"fmt"
+
+	"github.com/algorand/go-algorand/config"
+	"github.com/algorand/go-algorand/data/basics"
+)
+
+// ExecTxnFields captures the fields used by exec transactions.
+type ExecTxnFields struct {
+	_struct struct{} `codec:",omitempty,omitemptyarray"`
+
+	Inputs []string `codec:"inp`
+	Code string `codec:"cod"`
```

Comment:
Is this source code? Should it be []byte?

Reply:
Yes, soon enough.  Should be just string for now, to cheat with a file name.